### ISSUE TITLE
Crash when using Spanish Version 

### ISSUE THREAD

**Issue by dellis1972** [infinitespace-studios/VectorRumble#29]
We get a crash when using the Spanish version of the game.

```
Text contains characters that cannot be resolved by this SpriteFont.
Parameter name: text
```

This is because we are trying to use the text `¿Salir?` and the `¿` does not exist in the SpriteFont we are using. We need two update the SpriteFont to include special characters we need for other languages. (or have language specific sprite fonts?).

**Comment by dellis1972** [comment]
If you want to test this you can modify the Program.cs to use the following code.
```
#region Using Statements
using System;
using System.Globalization;
#endregion

namespace VectorRumble.Desktop
{
	static class Program
	{
		[STAThread]
		static void Main (string [] args)
		{
			System.Threading.Thread.CurrentThread.CurrentUICulture = CultureInfo.GetCultureInfo ("es-ES");
			System.Threading.Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo ("es-ES");
			using (var game = new VectorRumbleGame ()) {
				game.Run ();
			}
		}
	}
}
```
This will force mono/.net to use a specific Culture.

**Comment by CartBlanche** [comment]
👍

**Comment by CartBlanche** [comment]
fixed in #30
